examples/regression/4-tutorials/plot_cqr_tutorial.py

```diff
@@ -0,0 +1,411 @@
+"""
+==============================================
+Tutorial for conformalized quantile regression
+==============================================
+
+We will use the sklearn california housing dataset as the
+base for the comparison of the different methods available
+on MAPIE. Two classes will be used:
+:class:`mapie.regression.MapieRegressor` and
+:class:`mapie.quantile_regression.MapieQuantileRegressor`.
+
+The type of conformal methods are chosen with the parameter
+``method`` of :class:`mapie.regression.MapieRegressor` and
+the parameter ``cv`` is the strategy for cross-validation.
+In this method, to use a "leave-one-out" strategy, one would
+have to use `cv=-1` and a positive value would indicate the
+number of folds for a cross-validation strategy. Note that
+for the jackknife+ after boostrap, we need to use the
+class :class:`mapie.subsample.Subsample`.
```

Comment:
Briefly mention that CQR is a split-conformal method and we only split the training set into training/calibration sets ?

Reply:
done